README.md

```diff
@@ -65,6 +65,41 @@ One.sendInteraction("/interactionPath", { 'key' : 'value' });
 ```
 * See example of usage [here](https://github.com/thunderheadone/one-sdk-flutter/tree/master/examples/flutter_example/lib/main.dart#L128)
 
+To send an Interaction request and retrieve the response, call the following method:
+```dart
+One.sendInteraction("/interactionPath").then((response) {
+    print('Interaction response tid = ${response[oneResponseTidKey]}');
+    print('Interaction response Interaction path = ${response[oneResponseInteractionPathKey]}');
+    print('Interaction response optimization points = ${response[oneResponseOptimizationPointsKey]}');
+});
+```
```

Comment:
No example usage for this one?

Reply:
There is one in main.dart
See example of usage [here](https://github.com/thunderheadone/one-sdk-flutter/tree/master/examples/flutter_example/lib/main.dart#L65)